_pages/dev/ssr/server-side-rendering-optimization.md

```diff
@@ -86,53 +89,109 @@ The full list of parameters is described in the following section.
 
-- `concurrency` is a number that indicates how many concurrent requests are treated before defaulting to CSR. Usually, when the concurrency increases (more renders being done at the same time) the slower the response is for each request. To fine-tune it and keep response time reasonable, you can limit the maximum number of concurrent requests. All requests that are unable to render because of this will fall back to CSR.
+Recommendation: should be set according to your business needs, and for how long you want to keep the stale render in cache before evicting it. Should be set regardless if the `cache` option is enabled.
 
-  The default value is `20`.
+### renderKeyResolver
 
-- `ttl` (time to live) is a number that indicates the amount of time (in milliseconds) before a pre-rendered page is considered stale and needs to be rendered again.
+`renderKeyResolver` is a function that accepts `(req: Request) => string`, which maps the current request to a specific render key. The `renderKeyResolver` allows you to override the default key generator so that you can differentiate between rendered pages with custom keys.
 
-- `renderKeyResolver` is a function that accepts `(req: Request) => string`, which maps the current request to a specific render key. The `renderKeyResolver` allows you to override the default key generator so that you can differentiate between rendered pages with custom keys.
+By default, `renderKeyResolver` it uses the full request URL.
 
-  By default, `renderKeyResolver` uses `req.originalUrl`.
+Recommendation: it is recommended to use the default Spartacus rendering key resolver, especially in cases when your domain contains a base site information (e.g. `my.site.au` or `my.site.rs`).
 
-- `renderingStrategyResolver` is a function that accepts `(req: Request) => RenderingStrategy`, which allows you to define custom rendering strategies for each request. The available `RenderingStrategy` strategies work as follows:
+### renderingStrategyResolver
 
-  - `ALWAYS_CSR` always returns client-side rendered pages
-  - `DEFAULT` is the default behavior
-  - `ALWAYS_SSR` aLways returns server-side rendered pages
+`renderingStrategyResolver` is a function that accepts `(req: Request) => RenderingStrategy`, which allows you to define custom rendering strategies for each request. The available `RenderingStrategy` strategies work as follows:
 
-- `forcedSsrTimeout` is a number that indicates the time (in milliseconds) to wait for rendered pages when the render strategy for the request is set to `ALWAYS_SSR`. This prevents SSR rendering from blocking resources for too long if the server is under heavy load, or if the page contains errors.
+- `ALWAYS_CSR` always returns client-side rendered pages
+- `DEFAULT` is the default behavior
```

Comment:
What is the default behavior?

Reply:
Added `obeys the provided `SsrOptimizationOptions`.`